<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogUpravitSmazatUzivatele</class>
 <widget class="QDialog" name="DialogUpravitSmazatUzivatele">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>298</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Úprava/smazání uživatele</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Uživatelské jméno:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Celé jméno:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="vstupCeleJmeno">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Celé jméno</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="textAdmin">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Administrátor:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="volbaAdmin">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="textStareHeslo">
       <property name="text">
        <string>Staré heslo:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="textNoveHeslo">
       <property name="text">
        <string>Nové heslo:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="textNoveHeslo2">
       <property name="text">
        <string>Potvrzení hesla:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="vstupStareHeslo">
       <property name="toolTip">
        <string>Staré heslo</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="vstupNoveHeslo">
       <property name="toolTip">
        <string>Nové heslo</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="vstupNoveHeslo2">
       <property name="toolTip">
        <string>Potvrzení hesla</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QCheckBox" name="volbaSmazatUzivatele">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>smazat uživatele</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="seznamUzivJmeno">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="popisekUpozorneni">
     <property name="styleSheet">
      <string notr="true">color: red;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="tlacitkoZrusit">
       <property name="toolTip">
        <string>Zrušit (Esc)</string>
       </property>
       <property name="text">
        <string>Zrušit</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tlacitkoUlozit">
       <property name="toolTip">
        <string>Uložit (Ctrl+Enter)</string>
       </property>
       <property name="text">
        <string>Uložit</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>seznamUzivJmeno</tabstop>
  <tabstop>vstupCeleJmeno</tabstop>
  <tabstop>volbaAdmin</tabstop>
  <tabstop>vstupStareHeslo</tabstop>
  <tabstop>vstupNoveHeslo</tabstop>
  <tabstop>vstupNoveHeslo2</tabstop>
  <tabstop>volbaSmazatUzivatele</tabstop>
  <tabstop>tlacitkoZrusit</tabstop>
  <tabstop>tlacitkoUlozit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
